CONTRIBUTING.md

```diff
@@ -1,36 +1,56 @@
+A good place to start is to examine all of the [GitHub issues labeld "Wishlist"](https://github.com/NESCent/popgenInfo/labels/Wishlist). Please comment on the issue if you wish to contribute so that we know who is taking responsibility.
 
-## How do I submit a workflow?
+# How do I write a vignette?
 
-Overall, workflow submission will follow the guidelines in [GitHub flow](https://guides.github.com/introduction/flow/index.html). Please create a new branch for each workflow to be submitted.
+Contributed vignettes should be written in [R markdown](http://rmarkdown.rstudio.com/). A template can be found in [`TEMPLATE.Rmd`](https://github.com/nescent/popgenInfo/tree/master/TEMPLATE.Rmd) (See the [rendered example](TEMPLATE.html)). The vignette should include example analysis and explanations using data that can be readily accessed in the public domain, e.g., the data that comes as part of some packages, so that users can replicate the analysis themselves. If no public data is available to address a particular biological question that you want to write a vignette for, this data can be uploaded to this repository also.
 
-1. Set up a github account if you don't already have one, see [here](https://guides.github.com/activities/hello-world/) for a quick tutorial on using Github. Download an appropriate Git client for your computer, see [here](https://help.github.com/articles/set-up-git/), and set it up.
-2. Fork the [source repository](http://github.com/NESCent/popgenInfo) on Github using the `Fork` button. 
-3. Clone the repository to your machine using `git clone https://github.com/NESCent/popgenInfo.git` on the command line. The URL used here is in the HTTPS clone url section on the popgenInfo github page. Keep your clone synced up by following Step 3 of [this tutorial](https://help.github.com/articles/fork-a-repo/). 
-4. Then, move to the popgenInfo directory using `cd popgenInfo`.
+# How do I submit a vignette?
+
+Overall, vignette submission will follow the guidelines in [GitHub flow](https://guides.github.com/introduction/flow/index.html). Please create a new branch for each vignette to be submitted. 
+
+ 
+ 
+## Step-by-step Process for submission
+
+1. Set up a GitHub account if you don't already have one, see [here](https://guides.github.com/activities/hello-world/) for a quick tutorial on using GitHub. Download an appropriate Git client for your computer, see [here](https://help.github.com/articles/set-up-git/), and set it up.
+2. Fork the [source repository](http://github.com/NESCent/popgenInfo) on GitHub using the `Fork` button. 
+3. Clone the repository to your machine using `git clone https://github.com/YourAccount/popgenInfo.git` on the command line. The URL used here is in the HTTPS clone url section on the popgenInfo GitHub page. Keep your clone synced up by following Step 3 of [this tutorial](https://help.github.com/articles/fork-a-repo/). 
```

Comment:
I think we should indicate here as a short note that the HTTPS URL will be easier to follow for beginners (no need for setting up an ssh key), but that the SSH URL will work just as well (and in fact better if one has an SSH key). So I'm wondering whether we can't just refer to [Github's help](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork).